packages/app/ui/userAvatar.tsx

```diff
@@ -1,16 +1,19 @@
 import { User } from "app/api/graphql";
-import { Text, View, Image } from "app/design-system";
+import { Image, Text, View } from "app/design-system";
 import { compose, head, join, map, split, take, toUpper } from "ramda";
 
 type Props = {
   user: User;
+  big?: boolean;
```

Comment:
what about adding enum `size` as a prop instead? For now, there will be only two sizes: `default` and `large`. Then if we would like to support other sizes, it would be easier to add.

Reply:
Code using boolean is a bit cleaner, so until we really need more than two states, I would leave it like this